packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -1164,16 +1164,23 @@ class PopupMenuButtonState<T> extends State<PopupMenuButton<T>> {
 
     assert(debugCheckHasMaterialLocalizations(context));
 
-    if (widget.child != null)
-      return Tooltip(
-        message: widget.tooltip ?? MaterialLocalizations.of(context).showMenuTooltip,
-        child: InkWell(
-          onTap: widget.enabled ? showButtonMenu : null,
-          canRequestFocus: _canRequestFocus,
-          enableFeedback: enableFeedback,
-          child: widget.child,
-        ),
+    if (widget.child != null) {
```

Comment:
I'm not sure why you need this code in `popup_menu.dart` now: if the tooltip is empty, it already won't show anything. No point in having similar code in two places, especially when it makes the logic more complex.